test/unit/testDeceasedMarried.js

```diff
@@ -14,23 +14,37 @@ describe('DeceasedMarried', () => {
         });
     });
 
-    describe('handlePost()', () => {
-        let ctx;
-        let errors;
-        let formdata;
-        let session;
-        let hostname;
-        let featureToggles;
+    describe('getContextData()', () => {
+        it('should return the ctx with the deceased address and the screening_question feature toggle on', (done) => {
+            const req = {
+                sessionID: 'dummy_sessionId',
+                session: {form: {}, featureToggles: {screening_questions: true}},
+                body: {
+                    married: 'Yes'
+                }
+            };
+            const ctx = DeceasedMarried.getContextData(req);
+            expect(ctx).to.deep.equal({
+                married: 'Yes',
+                isToggleEnabled: true,
+                sessionID: 'dummy_sessionId'
+            });
+            done();
+        });
 
-        it('should return the ctx with the deceased married status and the screening_question feature toggle', (done) => {
-            ctx = {
-                married: 'Yes'
+        it('should return the ctx with the deceased address and the screening_question feature toggle off', (done) => {
```

Comment:
Please update the description for this test as it is for deceased married but it is referring to deceased address.